packages/components/src/checkbox-control/style.scss

```diff
@@ -1,3 +1,6 @@
+$input-size: 16px;
```

Comment:
Should these be declared somewhere more global (ie. [`_variables.scss`](https://github.com/WordPress/gutenberg/blob/master/assets/stylesheets/_variables.scss))?